bundles/org.openhab.binding.generacmobilelink/pom.xml

```diff
@@ -18,7 +18,7 @@
     <dependency>
       <groupId>org.jsoup</groupId>
       <artifactId>jsoup</artifactId>
-      <version>1.14.3</version>
+      <version>1.15.4</version>
```

Comment:
Can you upload this dependency to Artifactory @kaikreuzer?

Reply:
Done.